app/models/grda_warehouse/hud/project.rb

```diff
@@ -74,6 +73,8 @@ class Project < Base
     # project_coc has been deleted
     belongs_to :project_cocs_with_deleted, class_name: 'GrdaWarehouse::Hud::WithDeleted::ProjectCoc', primary_key: [:ProjectID, :data_source_id], foreign_key: [:ProjectID, :data_source_id], optional: true
 
+    include RailsDrivers::Extensions
```

Comment:
I moved this here to avoid the following error. I haven't run the full test suite yet so I don't feel confident this change isn't having any other negative impacts elsewhere.

`
Cannot have a has_many :through association 'GrdaWarehouse::Hud::Project#enrollment_location_histories' which goes through 'GrdaWarehouse::Hud::Project#enrollments' before the through association is defined. (ActiveRecord::HasManyThroughOrderError)
`

Reply:
That makes sense.  Sometimes we drop it at the bottom of the file, not sure if that's necessary here, or desirable.  Can you add a comment on line 75 to indicate it needs to come after `has_many :enrollments` so we don't accidentally move it back up?